src/internal_modules/roc_rtp/validator.cpp

```diff
@@ -66,6 +66,21 @@ bool Validator::check_(const packet::RTP& prev, const packet::RTP& next) const {
         sn_dist = -sn_dist;
     }
 
+    if (!!next.capture_timestamp && !!prev.capture_timestamp) {
+        const bool valid_capt_ts =
+            core::ns_equal(next.capture_timestamp, prev.capture_timestamp,
+                           config_.max_ts_jump)
+            && next.capture_timestamp > prev.capture_timestamp;
```

Comment:
1. max_ts_jump is in other units. I guess we need a new config field.
2. Validator is covered with tests. Please add a test for this new feature.
3. Let's move capture_ts check to the end? I think timestamp and seqnum checks are more informative in case of error (because they are in packet fields directly, not estmaited as capture ts) and it'd be better to make them first.